core/src/test/java/io/lionweb/utils/ChunkValidatorTest.java

```diff
@@ -241,6 +241,208 @@ public void testNodeWithPropertiesAndReferences() {
     assertTrue(result.isSuccessful());
   }
 
+  @Test
+  public void testDuplicateContainmentMetaPointers() {
+    SerializedClassifierInstance parent = createValidNode("parent1", null);
+    SerializedClassifierInstance child1 = createValidNode("child1", "parent1");
+    SerializedClassifierInstance child2 = createValidNode("child2", "parent1");
+
+    // Create two containments with the same metapointer
+    SerializedContainmentValue containment1 =
+        new SerializedContainmentValue(testMetaPointer, Arrays.asList("child1"));
+    SerializedContainmentValue containment2 =
+        new SerializedContainmentValue(testMetaPointer, Arrays.asList("child2"));
+
+    parent.addContainmentValue(containment1);
```

Comment:
Shouldn't this be `parent.unsafeAddContainmentValue`? I'd that method to exist, just as `unsafeAddPropertyValue`.

Reply:
You are right. I reviewed other similar methods to be renamed with the `unsafe` prefix